lib/utils/cli/cli_test.go

```diff
@@ -80,14 +84,23 @@ func (*S) TestUpdatesCommandLine(c *check.C) {
 			},
 			removeFlags: []string{"selinux"},
 		},
+		{
+			comment:    "Redact flags",
+			inputArgs:  []string{"install", `--token=token`, "--debug"},
```

Comment:
I'd like to test with a couple of commands since this change has an interesting twist of supporting multiple commands with a single set of replacement flags.